addons/mail/wizard/mail_compose_message.py

```diff
@@ -284,13 +284,10 @@ def get_mail_values(self, res_ids):
             BL_sudo = self.env['mail.blacklist'].sudo()
             blacklist = set(BL_sudo.search([]).mapped('email'))
             if blacklist:
-                [email_field] = self.env[self.model]._primary_email
-                targets = self.env[self.model].browse(res_ids).read([email_field])
+                targets = self.env[self.model].browse(res_ids).read(['email_normalized'])
```

Comment:
Are we sure the model from  `self.model` implements the address mixin and really have the `email_normalized` field ?

Reply:
yes, as we check : if mass_mail_mode and hasattr(self.env[self.model], "_primary_email"):